server/CONTRIBUTING.md

```diff
@@ -19,7 +19,8 @@
 #### Prerequisites
```

Comment:
Please add a link to `server/CONTRIBUTING.md` in `README`

Reply:
thx fixed